index.js

```diff
@@ -3,6 +3,16 @@ const markdownEscape = require('markdown-escape');
 const MAX_REGULAR_COMMITS_PER_RELEASE = 5;
 const semver = require('semver');
 
+const htmlEntity = (c) =>
+  ({
+    '<': '&lt;',
+    '>': '&gt;',
+  }[c]);
+
+const htmlEscape = (s) => s.replace(/[<>]/, htmlEntity);
```

Comment:
This will only replace the first occurrence unless you add the global flag:

```suggestion
const htmlEscape = (s) => s.replace(/[<>]/g, htmlEntity);
```